src/main/java/io/personium/core/PersoniumCoreException.java

```diff
@@ -1002,6 +1002,14 @@
          * {0} : Parse string
          */
         public static final PersoniumCoreException JSON_PARSE_ERROR = create("PR400-CM-0004");
+        /**
+         * Invalid URL authority.
+         * <p>
+         * {0} : Given authority
+         * {1} : Configured authority
+         */
+        public static final PersoniumCoreException INVALID_URL_AUTHORITY = create("PR400-CM-0005");
```

Comment:
I chose 400 because this error occurs whenever a client sends Host header value different from the one configured in the Unit Config, mistakenly or intentionally. 

Since an attacker can intentionally cause this error, I guess it would be problematic to use 500.

Reply:
Set this conversation "resolved" since we talked yesterday and agreed on this.